Content.Shared/WaterBalloon/WaterBalloonSystem.cs

```diff
@@ -0,0 +1,61 @@
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Chemistry.EntitySystems;
+using Content.Shared.Sprite;
```

Comment:
as i see you also don't need Sprite (like don't used at all) and Chemistry.Component (all methods are in system, along with the event)